src/metfish/commands.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import os
+import sys
+
+import pandas as pd
+
+from .utils import get_Pr
+
+def get_Pr_cli(argv=None):
+
+    desc = "Calculate an exact P(r) curve for a given structure"
+    epi = """By default, Dmax is calculated from the max distance foudn in the structure."""
```

Comment:
Just a small reminder. According to GitHub grammar checker, there are some typos. 
line12 foudn
line20 extist
 line27 overwritting
And also in the utils.py line16 structure.